.azure/pipelines/ci.yml

```diff
@@ -7,22 +7,716 @@ pr:
+      inputs:
+        sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-Release/
+        contents: '**'
+        targetFolder: $(Build.SourcesDirectory)/.deps/
+    - script: .\build.cmd
+              -ci
+              /p:SignType=$(_SignType)
+              $(BuildScriptArgs)
+              $(SharedFxArgs)
+              /p:SharedFxRID=win-x64
+              /bl:artifacts/logs/SharedFx-win-x64.binlog
+      env:
+        PB_PackageVersionPropsUrl: $(PB_PackageVersionPropsUrl)
+        PB_AssetRootUrl: $(PB_AssetRootUrl)
+        PB_RestoreSource: $(PB_RestoreSource)
+        PB_PublishBlobFeedKey: $(PB_PublishBlobFeedKey)
+      displayName: Build win-x64 SharedFX
+    - powershell: eng\scripts\KillProcesses.ps1
+      displayName: Kill processes
+      condition: always()
+    - script: .\build.cmd
+              -ci
+              /p:SignType=$(_SignType)
+              $(BuildScriptArgs)
+              $(SharedFxArgs)
+              /p:SharedFxRID=win-x86
+              /bl:artifacts/logs/SharedFx-win-x86.binlog
+      env:
+        PB_PackageVersionPropsUrl: $(PB_PackageVersionPropsUrl)
+        PB_AssetRootUrl: $(PB_AssetRootUrl)
+        PB_RestoreSource: $(PB_RestoreSource)
+        PB_PublishBlobFeedKey: $(PB_PublishBlobFeedKey)
+      displayName: Build win-x86 SharedFX
+    - powershell: eng\scripts\KillProcesses.ps1
+      displayName: Kill processes
+      condition: always()
+    - script: .\build.cmd
+              -ci
+              /p:SignType=$(_SignType)
+              $(BuildScriptArgs)
+              /p:SkipArtifactInfoTargets=true
+              /p:DisableSignCheck=true
+              /t:DoCodeSigning
+              /bl:artifacts/logs/CodeSign.binlog
+      env:
+        PB_PackageVersionPropsUrl: $(PB_PackageVersionPropsUrl)
+        PB_AssetRootUrl: $(PB_AssetRootUrl)
+        PB_RestoreSource: $(PB_RestoreSource)
+        PB_PublishBlobFeedKey: $(PB_PublishBlobFeedKey)
+      displayName: Code Sign
+    - powershell: eng\scripts\KillProcesses.ps1
+      displayName: Kill processes
+      condition: always()
+    - task: MicroBuildCleanup@1
+      displayName: Cleanup MicroBuild tasks
+      condition: always()
+    - task: PublishBuildArtifacts@1
+      displayName: Upload artifacts
+      condition: eq(variables['system.pullrequest.isfork'], false)
+      continueOnError: true
+      inputs:
+        pathtoPublish: artifacts/
+        artifactName: artifacts-Windows-SharedFx
+        artifactType: Container
+        parallel: true
+    - task: PublishBuildArtifacts@1
+      displayName: Upload dependencies.g.props
+      condition: eq(variables['system.pullrequest.isfork'], false)
+      continueOnError: true
+      inputs:
+        pathtoPublish: .deps/dependencies.g.props
+        artifactName: artifacts-dependencies-props
+        artifactType: Container
+        parallel: true
+
+  - job: MacOs_SharedFx
+    displayName: Build Osx-x64 SharedFx
+    dependsOn: Windows_Build
```

Comment:
Why is this depending on windows build?

Reply:
The Unix SharedFx builds depend on the windows product build in TeamCity as well - they download windows artifacts to `.deps` and reference them from there